envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/utils/ReactorUtils.kt

```diff
@@ -1,13 +1,21 @@
 package pl.allegro.tech.servicemesh.envoycontrol.utils
 
 import io.micrometer.core.instrument.MeterRegistry
+import org.reactivestreams.Subscription
 import org.slf4j.LoggerFactory
+import reactor.core.Disposable
 import reactor.core.Fuseable
 import reactor.core.Scannable
 import reactor.core.publisher.Flux
+import reactor.core.scheduler.Schedulers
+import java.time.Duration
+import java.util.concurrent.TimeUnit
 import kotlin.streams.asSequence
 
 private val logger = LoggerFactory.getLogger("pl.allegro.tech.servicemesh.envoycontrol.utils.ReactorUtils")
+private val scheduler by lazy { Schedulers.newSingle("reactor-utils-scheduler") }
```

Comment:
Problem with static methods, where to put state :)
How about adding this lazy scheduler to default parameter of `logSuppressedError` so someone can override it?

Reply:
done